.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+
+<!-- Please describe the issue in detail here, and for bug reports fill in the
+fields below -->
+
+### Reproducing code example:
+
+<!-- A short code example that reproduces the problem. It should be
+self-contained, i.e., possible to run as-is via 'python myproblem.py' -->
+
```

Comment:
I think providing
````
```python
import numpy as np
```
````

here would be helpful

Reply:
added